scripts/post_install_script.sh

```diff
@@ -211,26 +250,170 @@ modules:
     openmpi:
       environment:
         set:
-          SLURM_MPI_TYPE: pmix
-    netlib-scalapack:
-      suffixes:
-        '^openmpi': openmpi
-        '^mpich': mpich
-    ^python:
-      autoload:  direct
+          SLURM_MPI_TYPE: "pmix"
+          OMPI_MCA_btl_tcp_if_exclude: "lo,docker0,virbr0"
+    miniconda3:
+      environment:
+        set:
+          CONDA_PKGS_DIRS: ~/.conda/pkgs
+          CONDA_ENVS_PATH: ~/.conda/envs
   lmod:
     hierarchy:
       - mpi
 EOF
     cat ${spack_install_path}/etc/spack/modules.yaml
 
+	cat << EOF >> ${spack_install_path}/etc/spack/mirrors.yaml
+mirrors: { "aws-optimized": "s3://spack-mirrors/amzn2-e4s" }
+EOF
+    cat ${spack_install_path}/etc/spack/mirrors.yaml
+
+    mkdir -p ${spack_install_path}/var/spack/environments/aws
+	cat << EOF > ${spack_install_path}/var/spack/environments/aws/spack.yaml
+spack:
+  view: false
+  concretization: separately
+
+  mirrors: { "mirror": "s3://spack-mirrors/amzn2-e4s" }
+
+  packages:
```

Comment:
redundant?

Reply:
yes, I intend to drop the aws environment. This is residual from testing configurations. No need for a separate env.